compiler+runtime/src/cpp/clojure/core_native.cpp

```diff
@@ -265,6 +271,11 @@ namespace clojure::core_native
     __rt_ctx->compile_module(runtime::to_string(path)).expect_ok();
     return obj::nil::nil_const();
   }
+
+  static object_ptr eval(object_ptr const expr)
+  {
+    return expr;
```

Comment:
I'm struggling here with how to covert the expr to something that can be passed to [`evaluate::eval()`](https://github.com/jank-lang/jank/blob/main/compiler%2Bruntime/include/cpp/jank/evaluate.hpp#L19).

Reply:
Oh, you're right. Poor direction on my part. First you'll need to analyze the object. `runtime::context::eval_string` shows how this is done. Let's add a `runtime::context::eval` which takes the `object_ptr`. It'll then use `an_prc` to analyze that object in value position. Something like:

```cpp
static object_ptr eval(object_ptr const o)
{
  auto const expr(an_prc.analyze(o, analyze::expression_position::value));
  return evaluate::eval(expr.expect_ok());
}
```

Reply:
Done, thanks @jeaye!